sources/manifest.json

```diff
@@ -31,7 +31,7 @@
 	[
 		"storage",
 		"contextMenus",
-
```

Comment:
This may be a nitpick, but I like the blank line separating API permissions from host permissions, I think it adds clarity.